File: Logistics.Data/Inventory.cs
using System;
using System.Collections.Generic;

namespace Logistics.Data
{
    public class Inventory
    {
        public int Id { get; set; }
        public int PropertyRecordId { get; set; }
        public int UserId { get; set; }
        public DateTime DateStarted { get; set; }
        public string Remarks { get; set; }

        public InventoryVerification InventoryVerification { get; set; }
        public PropertyRecord PropertyRecord { get; set; }
        public User User { get; set; }

        public List<InventoryItem> InventoryItems { get; set; }
    }
}

File: Logistics.Web/ClientApp/src/app/models/inventory.ts
import { InventoryItem } from './inventory-item';
import { InventoryVerification } from './inventory-verification';
import { PropertyRecord } from './property-record';
import { User } from './user';

export class Inventory {
  id: number;
  propertyRecordId: number;
  userId: number;
  dateStarted: Date;
  remarks: string;
  inventoryVerification: InventoryVerification;
  propertyRecord: PropertyRecord;
  user: User;
  inventoryItems: InventoryItem[];
}
